libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/NativeSetRouteResult.kt

```diff
@@ -3,5 +3,6 @@ package com.mapbox.navigation.core.trip.session
 import com.mapbox.navigator.RouteAlternative
 
 internal data class NativeSetRouteResult(
-    val nativeAlternatives: List<RouteAlternative>? = null
+    val nativeAlternatives: List<RouteAlternative>? = null,
+    val error: String? = null,
```

Comment:
Can it have `nativeAlternatives` and `error` simultaneously?

Reply:
From the usage it seems that result is success **or** error 🤔

Reply:
internal sealed class candidate